scripts/test-functions-config.js

```diff
@@ -29,7 +29,6 @@ var preTest = function () {
   var dir = tmp.dirSync({ prefix: "cfgtest_" });
   tmpDir = dir.name;
   fs.copySync(projectDir, tmpDir);
-  api.setRefreshToken(configstore.get("tokens").refresh_token);
```

Comment:
is this intentional?

Reply:
It does exist in apiv2, if that matters. I'm not entirely sure though :)